src/node.rs

```diff
@@ -483,14 +491,25 @@ impl PbftNode {
                 let block_id_matches = block.previous_id == wb.get_block_id();
 
                 if !block_num_matches || !block_id_matches {
+                    debug!(
```

Comment:
There are a lot of logging updates in this commit; perhaps these should be to moved to the "logging improvement" commit that's a part of this PR to keep this commit clean.